simexpal/launch/common.py

```diff
@@ -250,20 +263,47 @@ def compile_manifest(run):
 		'timeout': float(exp.info.timeout) if exp.info.timeout is not None else exp.info.timeout,
 		'environ': environ,
 		'output': exp.info.output,
+		'output_extensions': exp.info.output_extensions,
+		'stdout': exp.info.stdout,
 		'workdir': exp.info.workdir
 	})
 
 def invoke_run(manifest):
-	# Create the output file. This signals that the run has been started.
+
+	def get_qualified_output_file(ext):
+		if ext not in set(manifest.output_extensions + ['out']):
+			raise RuntimeError(
+				f"Unexpected output extension for experiment '{manifest.experiment}': .{ext}\n"
+			)
+		return manifest.output_file_path(ext)
+
+	is_directed = False
```

Comment:
Rename to `redirect_stdout`.

Reply:
Or just set `output_path` to `None` to handle this case (is it ever used in the redirected case?).

Reply:
I did the latter

Reply:
The ``stdout_path`` is needed for the at-variable ``@OUTPUT@``. I was able to get rid of the extra boolean though.